### PR TITLE
Improve GitHub action

### PR DESCRIPTION
**Pull request by agners**
- Add YAML linter
- Drop GitHub registry login

### CODE REVIEW

File: .github/move.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Configuration for move-issues - https://github.com/dessant/move-issues
```
Comment (agners) [inline .github/move.yml:1]: I don't think this is active is it? So I can drop the whole file.

Comment (ludeeus) [reply .github/move.yml:1]: Drop it, we use native github move or ask the user to recreate issues in the correct repo.

File: .github/dependabot.yml [hunk 2]
```diff
@@ -1,8 +1,9 @@
...
-  directory: "/"
-  schedule:
-    interval: daily
-    time: "06:00"
-  open-pull-requests-limit: 10
\ No newline at end of file
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: daily
```
Comment (ludeeus) [inline .github/dependabot.yml:7]:
```suggestion
      interval: weekly
```
Let's do weekly for these

File: deconz/config.yaml [hunk 3]
```diff
@@ -1,7 +1,9 @@
+---
 version: 6.12.0
 slug: deconz
 name: deCONZ
-description: Control a Zigbee network with ConBee or RaspBee by Dresden Elektronik
+description: >
```
Comment (ludeeus) [inline deconz/config.yaml:5]:
```suggestion
description: >-
```
Use `>-` to strip whitespace for these (same goes for all others like it)

File: letsencrypt/config.yaml [hunk 4]
```diff
@@ -66,7 +67,11 @@ schema:
     ovh_consumer_key: str?
     ovh_endpoint: str?
     propagation_seconds: int(60,3600)?
-    provider: list(dns-cloudflare|dns-cloudxns|dns-digitalocean|dns-directadmin|dns-dnsimple|dns-dnsmadeeasy|dns-gehirn|dns-google|dns-linode|dns-luadns|dns-njalla|dns-nsone|dns-ovh|dns-rfc2136|dns-route53|dns-sakuracloud|dns-netcup|dns-gandi|dns-transip)?
+    provider: "list(dns-cloudflare|dns-cloudxns|dns-digitalocean|\
```
Comment (ludeeus) [inline letsencrypt/config.yaml:70]: Can you use `>-` with multiline here to make it more readable?

Comment (agners) [reply letsencrypt/config.yaml:70]: I don't think this works, `>-` only strips newlines at the end (according to https://yaml-multiline.info/)

Comment (ludeeus) [reply letsencrypt/config.yaml:70]: `|-` should work

**Review by ludeeus — APPROVED** [review]

**Pull request merged**